mne/utils/docs.py

```diff
@@ -1389,7 +1390,19 @@
 """
 docdict['subject'] = """
 subject : str
-    The FreeSurfer subject name.
+    The FreeSurfer subject name (will be used to set the FreeSurfer
+    environment variable ``SUBJECT``).
```

Comment:
Usually setting the env var is not actually done -- we only do this / set the env var when making actual calls to FreeSurfer functions. So I think this is not quite right...

It acts like the SUBJECT env var does for FreeSurfer, but I don't think it's worth mentioning here

Reply:
I just mentioned it because other docstrings mention(ed), "corresponding to the FreeSurfer SUBJECT variable".

But I get your point and will change this